### PR TITLE
Copy stack trace to clipboard

### PR DESCRIPTION
**Pull request by zivkaziv**
Associated Issue: #[2746](https://github.com/devtools-html/debugger.html/issues/2746)
 
### Summary of Changes
You can now copy the call stack by right clicking on the call stack.
The output look like:

> initialize  todo-view.js#34)
child  (backbone.js#1566)
addOne  (app-view.js#79)
forEach  (underscore.js#79)
method  (backbone.js#966)
addAll  (app-view.js#86)
fetch  (backbone.js#870)

* Adding **formatCopyName** to the util/frame.js
This method is for creating the copy name standard 

### Test Plan
Example test plan:
- [x] Open the debugger
- [x] Set break point
- [x] Perform an action that will run the code and stop at the break point
- [x] On the Call Stack frame press right click
- [x] Select "Copy Stack Trace" from the context menu
- [x] Paste the clipboard in to notepad and compare between the trace on the debugger to the pasted one

### Screenshots/Videos
[Link to gif](http://recordit.co/lvhctJTNbR)

### CODE REVIEW

File: src/components/SecondaryPanes/Frames.js [hunk 1]
```diff
@@ -107,10 +113,23 @@ class Frames extends Component {
       menuOptions.push(copySourceUrl);
     }
 
+    if (source) {
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/Frames.js]: I don't think we need an IF statement here

File: src/components/SecondaryPanes/Frames.js [hunk 2]
```diff
@@ -107,10 +113,23 @@ class Frames extends Component {
...
+      };
+
+      menuOptions.push(copyStackTrace);
+    }
+
     showMenu(event, menuOptions);
   }
 
-  renderFrame(frame: LocalFrame) {
+  renderFrame(frame: LocalFrame, frames) {
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/Frames.js]: `renderFrame(frame: LocalFrame, frames: LocalFrame[]) `

File: src/utils/frame.js [hunk 3]
```diff
@@ -97,3 +98,11 @@ export function formatDisplayName(frame: Frame) {
...
   return endTruncateStr(displayName, 25);
 }
+
+export function formatCopyName(frame: Frame) {
+  const displayName = formatDisplayName(frame);
+  const fileName = getFilename(frame.source);
+  const frameLocation = frame.location.line;
+
+  return `${displayName}  ${fileName}#${frameLocation}`;
+}
```
Comment (jasonLaster) [inline src/utils/frame.js]: lets keep this on the top of the component `Frame` because it is kinda like rendering

File: src/utils/frame.js [hunk 4]
```diff
@@ -97,3 +98,11 @@ export function formatDisplayName(frame: Frame) {
   displayName = simplifyDisplayName(displayName);
   return endTruncateStr(displayName, 25);
 }
+
+export function formatCopyName(frame: Frame) {
+  const displayName = formatDisplayName(frame);
+  const fileName = getFilename(frame.source);
+  const frameLocation = frame.location.line;
+
+  return `${displayName}  ${fileName}#${frameLocation}`;
```
Comment (jasonLaster) [inline src/utils/frame.js]: lets change this to `${displayName} (${fileName}#${frameLocation})` to be consistent with chrome and visually group the file name

**Comment by zivkaziv** [comment]
Thanks for the review.
I've fixed all the issues below, but I'm getting error on "yarn flow"..
I'm working to fix it and hopefully push the new code soon

**Comment by jasonLaster** [comment]
@zivkaziv I made a couple tweaks to accomodate flow, formatting, and add a test. Let me know what you think.

**Comment by zivkaziv** [comment]
@jasonLaster your changes looks great, and now the yarn flow is OK (I still not sure what I did wrong):)
On of your comments was to move the method formatCopyName (utils/frame.js) to Frame component.
Do you still want me to do so? Or let's keep it as is?

Thanks again..

**Comment by jasonLaster** [comment]
I think it's fine as a util. It looks like some of the other utils in that
file :)

**Comment by zivkaziv** [comment]
In this case, I think we are ready to roll...

**Pull request merged**